### DISCUSSION TITLE
Proposed Enhancements for this Project

### DISCUSSION THREAD

**Discussion by ysrazsingh** [KevinVandy/material-react-table#1044] [General]
**I'm excited to contribute to your project and would like to propose some valuable functionalities based on my experience.**

**I've successfully implemented a similar table structure in a previous project, incorporating custom features tailored to client needs.** 

These features included:

Inline Editing with Confirmation and Server-Side Data Saving: This allows users to directly edit data within the table, such as names, emails, and dropdown selections. All edits are confirmed before finalization and securely saved to your database.

This approach offers several benefits:

**Improved User Experience** : Inline editing streamlines data entry and updates, making the process more user-friendly.
**Enhanced Data Accuracy** : Confirmation prompts minimize the risk of errors during data modification.
**Real-Time Data Management** : Automatic server-side saving ensures your database remains up-to-date with the latest information.

I'm confident these features can significantly improve the efficiency and user experience of your project. I'm happy to discuss this further and tailor these functionalities to your specific requirements.

**Video Reference** : https://mega.nz/file/tYkSSDYT#NK8h_u1CONpewKTrl_Mq0-msQdQJyb0s_GlRhT6RZgk

Thank You

**Comment by ysrazsingh** [comment]
@KevinVandy , Please review it once

**Reply by KevinVandy** [reply]
You can already do all of this with MRT. What APIs would even need to be added to make it easier? All input events are already exposed.

**Reply by ysrazsingh** [reply]
Hi @KevinVandy 

Sorry, I wasn't sure what you meant above. I just started working on Inline Cell Editing V2 today while on the MRT.
Could you please review this and let me know if I should proceed further, or not?
_https://mega.nz/file/1BcVCRKL#SxGKDaNnNIxniuLoRA73yXvIBZWkhr0AkJLZbORRRnU_

Thanks!

**Reply by ysrazsingh** [reply]
@KevinVandy

**Comment by ysrazsingh** [comment]
@KevinVandy, Please let me know if you are available

**Reply by KevinVandy** [reply]
For what?

**Reply by ysrazsingh** [reply]
Please Confirm this and review video once. 

I just started working on Inline Cell Editing V2 today while on the MRT.
Could you please review this and let me know if I should proceed further, or not?
https://mega.nz/file/1BcVCRKL#SxGKDaNnNIxniuLoRA73yXvIBZWkhr0AkJLZbORRRnU

Thanks!

**Reply by KevinVandy** [reply]
I do not really plan on changing the official editing CRUD examples much soon. If you are implementing customizations for yourself using all of the MRT and TanStack Table APIs that are available to you, that's great, and what you are supposed to do.

But I have no idea what you are actually proposing here. The video you shared is fine. If you want to make a code sandbox showing others what you made, that's great. I've actually been thinking about making a "Community Examples" page for a long time, but didn't know how many people might want to contribute.

If you are wanting new APIs or options, then actually propose them instead of showing a video. I'm pretty sure you can build everything in your video with how MRT is today, correct? If not, share the limitations and what could be added to MRT to make it easier.
